web/src/features/fireWeather/components/tables/StationComparisonTable.tsx

```diff
@@ -102,26 +102,33 @@ const StationComparisonTable = (props: Props) => {
               <TableRow>
                 <TableCell></TableCell>
                 <TableCell className={classes.darkColumnHeader} colSpan={5}>
-                  Temperature
+                  Temperature &deg;C
```

Comment:
Other places use the unicode version of degree, should we be using the html symbol codes everywhere?

Reply:
I honestly don't know what's best. I find that &deg; reads easier than just some obscure unicode character.

Reply:
Oh - or do you mean it's straight up got the degree symbol in there? I kind of dislike that as well - since I suspect there's more than one unicode character that looks like a degree symbol - so how do we know which one it is?

Reply:
I also do not know, but whatever way is chosen I think should be consistent. Some places have the straight up degree symbol, other places like here have the HTML code.

Reply:
I think the bit's where it's straight up a degree symbol isn't HTML - I'll check.

Reply:
```suggestion
                  Temperature (&deg;C)
```

Reply:
@conbrad / @andrea-williams and ideas on how to standardize on &deg; - we can't use the degree character straight up in react, because it's not valid html - how do I make the label: 'blah' section take label: 'blah &deg;' ?

Reply:
I think `&deg;` is the easiest-to-read HTML-compliant option. There's no way I'm going to memorize unicode numbers.

Reply:
Oh sorry  - things got confused with escaping. In places we are doing &amp;deg; -  in other places we straight up just use &deg; (see web/src/features/fireWeather/components/tables/ObservationTable.tsx ; we have code like `label: 'Dew Point (°C)',` - I don't know how to make it consistent so we use &amp;deg; in all places.

Reply:
@Sybrand have you tried any of these options?
https://stackoverflow.com/a/13094667

Reply:
No dice. None apply.

Reply:
Strange. Well we can defer this I think if it's taking too much time / not critical.